### ISSUE TITLE
Game always asks if you want to save, even if you just did

### ISSUE THREAD

**Issue by deurklink** [OpenRCT2/OpenRCT2#9904]
**OS:** Windows 10
**Version:** 0.2.3
**Commit/Build:** a094c5b

If you want to close the game, it normally would not ask if you want to save if you just saved. Now it always does.

**Comment by ZehMatt** [comment]
Is it really an issue? I refactored this to always pop up when in multiplayer.

**Comment by deurklink** [comment]
I think it is silly to ask if you want to save if you just did in the last minute.. Even in multiplayer.

**Comment by ZehMatt** [comment]
I disagree, the game currently has no understanding of when its worth saving other than the time elapsed between last save. I don't quite get the issue, this will only show up when you are about to exit, this also acts as a fail safe to not accidentally exit and having to reconnect, also its super easy to release the mouse button given that we have to hold it on some buttons.

**Comment by deurklink** [comment]
Ok, well I reported it because I thought it was a bug. Personally, I find it annoying because I always save before closing the game. It's not really a big issue, just an annoyance. If someone just saved it doesn't really matter if they accidentally close the game, does it? I also wonder how often it really happens.

Another thing, If I'm not mistaken, I believe the game does not ask if you want to save if you choose New Game (unless that was changed, I don't have access to the game at the moment). It would probably be a good thing to ask if the user wants to save there.

**Comment by ZehMatt** [comment]
Sounds like a good time to change your exit strategy by using the prompt to save now 😛. Regarding "New Game" that is a good point actually

**Comment by janisozaur** [comment]
Given that the information regarding current tick, related PRNG seed, peep state and viewport config are included in the sv6, technically the _only_ way to not modify the park is to have it loaded in paused state, not move the viewport at all and have the "export objects" option match the park in question.

**Comment by IntelOrca** [comment]
That isn't true, only if you do a game action does it count as truly modifying the park.
